lib/challenge.js

```diff
@@ -34,6 +51,10 @@ var pet_owner;
 
 // Answer Starts Here
 
+var pet_owner = {
+  name: "Jeff Meyer"
+  email: "160 Quincy Shore Drive, Quincy MA 02171"
+};
```

Comment:
This object literal is mostly well-formed and has both requested attributes. However, the missing comma that should follow the first key-value pair (`name: "Jeff Meyer"`) will throw a syntax error. I don't see this mistake anywhere else in your submission, so we'll call it a typo. 1